personality/argus/config.pan

```diff
@@ -38,58 +38,67 @@ variable PEP_HOME ?= ARGUS_LOCATION + '/pepd';
 #-----------------------------------------------------------------------------
 
 variable PAP_ENABLED = {
-  if (PAP_HOST == FULL_HOSTNAME) {
-    true;
-  } else {
-    false;
-  }
+    if (PAP_HOST == FULL_HOSTNAME) {
+        true;
+    } else {
+        false;
+    }
 };
 
-include { if (PAP_ENABLED) 'personality/argus/pap' };
+include if (PAP_ENABLED) 'personality/argus/pap';
```

Comment:
if PAP_ENABLED is only used here, do the test in the `if()`

Reply:
It is also use later.